app/src/main/java/com/google/android/gms/samples/wallet/CheckoutActivity.java

```diff
@@ -136,6 +138,21 @@ private void handlePaymentSuccess(PaymentData paymentData) {
         // getPaymentMethodToken will only return null if PaymentMethodTokenizationParameters was
         // not set in the PaymentRequest.
         if (token != null) {
+            // If the gateway is set to example, no payment information is returned - instead, the
+            // token will only consist of "examplePaymentMethodToken".
+            if (token.getToken().equals("examplePaymentMethodToken")) {
+                AlertDialog alertDialog = new AlertDialog.Builder(this)
+                        .setTitle("Warning")
+                        .setMessage("Gateway name set to \"example\" - please modify " +
+                                "Constants.java and replace it with your own gateway.")
+                        .setPositiveButton("OK", new DialogInterface.OnClickListener() {
```

Comment:
nit: I think you can just pass a `null` listener here if you don't want to do anything.